admin/src/main/scala/za/co/absa/spline/admin/commands.scala

```diff
@@ -67,3 +70,12 @@ case class DBExec(
 
   def addAction(action: AuxiliaryDBAction): DBExec = copy(actions = actions :+ action)
 }
+
+case class DBPrune(
+                    override val dbUrl: Url = null,
+                    retentionPeriod: Option[Duration] = None,
+                    thresholdDate: Option[ZonedDateTime] = None
+                  ) extends DBCommand {
```

Comment:
```suggestion
case class DBPrune(
  override val dbUrl: Url = null,
  retentionPeriod: Option[Duration] = None,
  thresholdDate: Option[ZonedDateTime] = None
) extends DBCommand {
```

We usually stick to Scala Style Guide, where the recommended indentation is constant
https://docs.scala-lang.org/style/declarations.html#classes

Reply:
Thanks, changed as suggested.